<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winMain</class>
 <widget class="QMainWindow" name="winMain">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Black hole simulation</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="wCentral">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="2" column="1">
     <widget class="QWidget" name="wRenderOpts" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="twRenderOpts">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabRenderOptsWorld">
          <attribute name="title">
           <string>World</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QScrollArea" name="scarOpts">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scarwWorldOpts">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>145</width>
                <height>268</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="lbRS">
                 <property name="text">
                  <string>Schwarzschild radius</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledRS">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbBgTex">
                 <property name="text">
                  <string>Background texture</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbBgFromPreset">
                 <property name="text">
                  <string>From preset</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbBgPreset1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbBgFromFile">
                 <property name="text">
                  <string>From file</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="wBgFromFile" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLineEdit" name="ledBgPath"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pbChooseBgPath">
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="spWorldOpts">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>322</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabRenderOptsObject">
          <attribute name="title">
           <string>Cluster</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QStackedWidget" name="stwObjectOpts">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="pageEc">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QScrollArea" name="scarEcOpts">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scarwEcOpts">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>185</width>
                    <height>531</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QLabel" name="lbEcNumPt">
                     <property name="text">
                      <string>Number of particles</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="ledEcNumPt"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="lbEcNormVec">
                     <property name="text">
                      <string>Normal vector</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="wEcNormVec" native="true">
                     <layout class="QFormLayout" name="formLayout">
                      <property name="horizontalSpacing">
                       <number>6</number>
                      </property>
                      <property name="verticalSpacing">
                       <number>0</number>
                      </property>
                      <property name="margin">
                       <number>0</number>
                      </property>
                      <item row="1" column="0">
                       <widget class="QLabel" name="lbEcNormVecY">
                        <property name="text">
                         <string>y</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="1">
                       <widget class="QLineEdit" name="ledEcNormVecY"/>
                      </item>
                      <item row="2" column="0">
                       <widget class="QLabel" name="lbEcNormVecZ">
                        <property name="text">
                         <string>z</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QLineEdit" name="ledEcNormVecX"/>
                      </item>
                      <item row="2" column="1">
                       <widget class="QLineEdit" name="ledEcNormVecZ"/>
                      </item>
                      <item row="0" column="0">
                       <widget class="QLabel" name="lbEcNormVecX">
                        <property name="text">
                         <string>x</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="lbEcSemiaxes">
                     <property name="text">
                      <string>Semiaxes</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="ledEcA"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="ledEcB"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="lbEcSize">
                     <property name="text">
                      <string>Size</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="ledEcDR"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="ledEcDZ"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="lbEcColor">
                     <property name="text">
                      <string>Color</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="wEcColor" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <item>
                       <widget class="QLabel" name="lbEcColorDisplay">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pbEcChooseColor">
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="spacerEc">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>173</width>
                       <height>213</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pbGenCluster">
                     <property name="text">
                      <string>Create Elliptic Cluster</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_4">
              <layout class="QVBoxLayout" name="verticalLayout_6"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="RenderWidget" name="wglRender" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>225</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionUser_Preferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionUser_Preferences">
   <property name="text">
    <string>User Preferences</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>renderwidget.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>twRenderOpts</tabstop>
  <tabstop>scarOpts</tabstop>
  <tabstop>ledRS</tabstop>
  <tabstop>rbBgFromPreset</tabstop>
  <tabstop>rbBgFromFile</tabstop>
  <tabstop>ledBgPath</tabstop>
  <tabstop>pbChooseBgPath</tabstop>
  <tabstop>scarEcOpts</tabstop>
  <tabstop>ledEcNumPt</tabstop>
  <tabstop>ledEcNormVecX</tabstop>
  <tabstop>ledEcNormVecY</tabstop>
  <tabstop>ledEcNormVecZ</tabstop>
  <tabstop>ledEcA</tabstop>
  <tabstop>ledEcB</tabstop>
  <tabstop>ledEcDR</tabstop>
  <tabstop>ledEcDZ</tabstop>
  <tabstop>pbEcChooseColor</tabstop>
  <tabstop>pbGenCluster</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
